### DISCUSSION TITLE
Can't open url: intent://arvr.google.com while trying to access3d file in the ar

### DISCUSSION THREAD

**Discussion by viniith21** [google/model-viewer#4561] [Q&A]
<div class="container">
      <model-viewer
        id="model"
        alt="3D MODEL"
        src=""
        auto-rotate
        shadow-intensity="1"
        camera-controls
        touch-action="pan-y">
      </model-viewer>
      <div class="model-info">
        <label for="title">Model Title</label>
        <p id="title">Loading...</p>
        <label for="description">Model Description</label>
        <p id="description"></p>
      </div>
    </div>

    <script>
      const urlParams = new URLSearchParams(window.location.search);
      const modelUrl = urlParams.get('model');
      const metadataString = urlParams.get('metadata');

      if (modelUrl) {
        const modelViewer = document.querySelector('model-viewer');
        modelViewer.src = modelUrl;
      }

      if (metadataString) {
        const metadata = JSON.parse(decodeURIComponent(metadataString));
        const title = metadata.title || 'No Title';
        const description = metadata.description || 'No Description';

        const titleElement = document.getElementById('title');
        titleElement.textContent = title;

        const descriptionElement = document.getElementById('description');
        descriptionElement.textContent = description;
      }
    </script>
  </body>
</html>
i am trying to access the 3D file from the server i got the Preview of the 3d but when  i try to access it in the ar it shows me this error 
Can't open url: intent://arvr.google.com/scene-viewer/1.0?mode=ar_preferred&disable_occlusion=true&file=https%3A%2F%2Fcdn-luma.com%2Fede55bce1c8f80f445e9ada9adc27ff4ec7b8c463f370c14c5e7ae9302a29d06%2FLord_Krishna_textured_mesh_glb.glb#Intent;scheme=https;package=com.google.ar.core;action=android.intent.action.VIEW;S.browser_fallback_url=https%3A%2F%2Fluma-modal-viewer.netlify.app%2F%3Fmodel%3Dhttps%3A%2F%2Fcdn-luma.com%2Fede55bce1c8f80f445e9ada9adc27ff4ec7b8c463f370c14c5e7ae9302a29d06%2FLord_Krishna_textured_mesh_glb.glb%26metadata%3D%7B%2522title%2522%3A%2522Lord%2520Krishna%2522%2C%2522status%2522%3A%2522complete%2522%7D%23model-viewer-no-ar-fallback;end;
please help me out how to solve this issue

**Comment by elalish** [comment]
Please fill in the issue template instead of deleting it. We need that information in order to help you.

**Reply by viniith21** [reply]
i didnt get you

**Reply by elalish** [reply]
Oh, sorry - I thought you opened this as an issue, but I see now it's a discussion. Maybe we need to put in the same issue template in the discussion section. Can you give us a link to your live website so we can see for ourselves? And what device/browser? Does it work correctly on any others?

**Reply by viniith21** [reply]
actually i am working on  an react-native application where i am using webView Screen to connect the html page on that page i am defining the model-viewer and passing data in that so my model is getting load but when iam clicking on ar it is showing this error

**Reply by viniith21** [reply]
Can i paste the code so that u can get ? all the threee file code ?

**Reply by elalish** [reply]
No, you'll need to make some kind of a minimal repro in a [Glitch](https://glitch.com/) or similar.

**Reply by viniith21** [reply]
i have already a repo in my github should i provide link?

**Reply by elalish** [reply]
To a live website, not just your source code.

**Reply by viniith21** [reply]
actually it is live but the data i am sending to website is from the Application the data which model-viewer is getting from the Application and in that also i am fetching the 3D link from another Company database
